src/test/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/postponementrequest/PostponementRequestMidEventHandlerTest.java

```diff
@@ -61,7 +62,8 @@ public void setUp() {
         handler = new PostponementRequestMidEventHandler(generateFile, templateId);
 
         Hearing hearing = getHearing(1);
-        List<Hearing> hearings  = List.of(hearing);
+        List<Hearing> hearings = new ArrayList<>(Collections.singleton(hearing));
```

Comment:
How come you changed this?

Reply:
Because List.of() makes the list immutable.  getLatestHearing() method sorts the list in reverse order, which cannot be done in an immutable list